.changeset/strong-swans-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': minor
```

Comment:
Ah shiet, this should be `patch` as we've gotten rid of any breaking changes and we're on `0.x`.